charts/datafold/values.yaml

```diff
@@ -152,7 +152,7 @@ worker:
   worker:
     queues: "celery,api,ci,interactive,freshpaint,alerts"
     tasks: 200
-    memory: 850000
+    memory: 1000000
```

Comment:
Maybe set it as `memory: "1000000"`, see https://datafold.slack.com/archives/C01KRSBJTEC/p1723634134133389

Reply:
The issue was caused by having '1G" before this one. The current setting works ok. The setting is converted to this string in the helm-charts , so kubernetes sees the stringified value: 

  value: "{{ .Values.worker.memory }}"